docker-compose.yaml

```diff
@@ -10,14 +10,15 @@ services:
     image: ankerctl/ankerctl:latest
     container_name: ankerctl
     restart: unless-stopped
+    pull_policy: build
     build: .
     environment:
       - FLASK_PORT=4470
       - FLASK_HOST=0.0.0.0
     volumes:
       - ankerctl_vol:/root/.config/ankerctl
     ports:
-      - 127.0.0.1:4470:4470
+      - 4470:4470
```

Comment:
There may be a security concern here. We don't want to publish on all ips because what will open up the flask server to anyone on the users network. 

127.0.0.1 was chosen intentionally to prevent that scenario

Reply:
Yep, what @spuder said. Maybe there's some way to make this configurable, that we can document? Or some way to override it when starting the docker? I'm open to suggestions, but I'm not a fan of just exposing the service to the world, since there's no authentication.